client/packages/system/src/Encounter/DetailView/VaccinationsTab.tsx

```diff
@@ -2,36 +2,51 @@ import React from 'react';
 import { VaccinationModal } from '../../Vaccination';
 import { useEditModal } from '@common/hooks';
 import { Clinician } from '../../Clinician';
+import { VaccineCardTable } from '../../Patient/VaccinationCards/VaccineCardTable';
 
 export const VaccinationsTab = ({
```

Comment:
I think you mentioned this already - we should probably move this, and put it with VaccineCardTable component. The "Tab" name doesn't make much sense, it's just making rendering the card with the modal (maybe we call this component VaccineCard?)

If it was doing anything specific here to be a tab in the patient view, this separate component would make sense, but I don't think it's adding any value on that front anymore :)

Reply:
Yeah, I think they all could do with a bit of re-organisation. Will quickly look at it with you at tomorrow morn's stand-up.